frontend/src/components/Image/ImageList.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import ImageRemovable from './ImageRemovable';
+import {css} from 'aphrodite';
+import styles from './styles';
+
+export default class ImageList extends React.Component {
+  removeImage = (ndx) => {
+    this.props.removeImage( ndx);
+  };
+  render = () => {
+    const image_list = this.props.images.map( (img, i) => {
+      let src;
+      if( typeof img === "string"){
+        src = img;
+      } else {
+        src = img.preview;
+      }
+      return (
+        <ImageRemovable src={src} key={i} ndx={i} removeImage={this.removeImage} />
+      );
+    });
+    return (
+      <div className={css(styles.image_box)} >
```

Comment:
You seem to have an inconsistent style. For example you do `this.props.images.map( (img, i))` but here you have no space between `css(styles.image_box)`

Reply:
for single param, no spaces, for multiple always space after comma.

Reply:
i was referring to the space after `map( HERE` whereas when you do `css(styles...` you don't write it like `css( styles....`

This is inconsistent. Either you do `map((img, i))` and `css(styles...` or you do `map( (img, i))` and `css(styles....` But you're currently just mixing and matching where you like :P . I agree after commas always spaces. e.g `banter, hello, true` but going `( banter, hello, true)` just seem counterintuitive to me.

Reply:
yes I see your point of view, but crockford says always space after brackets, I think he allows no space for the one param case. I have changed them ... let's get this merged I have two more pr's in waiting and about to start next.

The reason for the inconsistency is difference between my natural style and trying to change to a particular project. The fcc linter is remarkably tight, but we don't have time to write a linter like that me thinx. I still get lint errors for things I shouldn't, e.g. `state = {};`